CRM/Core/EntityTokens.php

```diff
@@ -99,7 +99,7 @@ public function evaluateToken(TokenRow $row, $entity, $field, $prefetch = NULL)
     $fieldValue = $this->getFieldValue($row, $field);
     if (is_array($fieldValue)) {
       // eg. role_id for participant would be an array here.
-      $fieldValue = implode(',', $fieldValue);
+      $fieldValue = implode(', ', $fieldValue);
```

Comment:
the non-token version of participant role was doing a space & the token version wasn't. Either could be argued for but If tests pass I think this is OK -